### DISCUSSION TITLE
Walt.Id Web Wallet

### DISCUSSION THREAD

**Discussion by magnusk85** [walt-id/.github#65] [Help] [answered]
Hello dear walt.id team,

i tried to use your web wallet. I ran your web wallet via executing the Docker compose file. After i created a demo account and was logged in, i tried to create one of these verifiable credentials. But the created (and accepted) credential does not appear in my wallet. I also recognized that in the wallet backend container there is a connection refused error when i m creating the credential. Can you tell me if i m doing something wrong or is there a bug in the code?

Thanks,

Magnus

**Comment by mikeplotean** [comment, marked as answer]
Hi, indeed there was an issue with the backend, but it is fixed now. Could you please provide more details, or maybe the error logs?

**Comment by magnusk85** [comment]
Hi Mike,
i'm sorry. The error i recognized was in the walletkit. It is a ConnectionRefused error. I attached the Logfile. Thank you for your help.


![grafik](h
[walletkit.log.txt](https://github.com/walt-id/.github/files/12557441/walletkit.log.txt)
ttps://github.com/walt-id/.github/assets/106743028/3b679e15-e63b-453b-9b58-8a9d83da5cd4)

**Reply by mikeplotean** [reply]
Hi, please pull the latest [waltid-web-wallet](https://github.com/walt-id/waltid-web-wallet). There's been some updates in the services configuration that might help in this case.
